nemo_skills/inference/prompt/nemotron/mmlu.yaml

```diff
@@ -0,0 +1,20 @@
+system: ""
+
+user: |-
+    Answer the following multiple choice question. The last line of your response should be of the following format: 'Answer: \boxed{{$LETTER}}' (without quotes) where LETTER is one of ABCD. Think step by step before answering.
```

Comment:
We can actually remove the \boxed{} thing from mmlu prompts now - can you please add a new parameter here https://github.com/Kipok/NeMo-Skills/blob/main/nemo_skills/evaluation/settings.py#L61 with something like this

```
'mmlu': '++eval_config.extract_from_boxed=False ++eval_config.extract_regex="Answer: (.+)$"'
```

The boxed format is obviously not what's typically used here and we should probably not be using it as well